test/property.test.js

```diff
@@ -125,6 +292,20 @@ describe('loopback:property generator', function() {
     });
   });
 
+  it('fails to create with unsupported type', function (done) {
+    var propertyGenerator = givenPropertyGenerator();
+    helpers.mockPrompt(propertyGenerator, {
+      model: 'Car',
+      name: 'something',
+      type: 'unknown',
+      required: 'true',
+      defaultValue: 'X'
+    });
+
+    expect(propertyGenerator.run).to.throw(Error);
+    done();
```

Comment:
Since this tests completes in the same tick, there is no need to use `done` callback. Please remove this line and remove `done` from function arguments.